hera_qm/xrfi.py

```diff
@@ -1732,6 +1751,21 @@ def day_threshold_run(data_files, history, nsig_f=7., nsig_t=7.,
     files = [glob.glob(d + '/*.flags2.h5')[0] for d in xrfi_dirs]
     uvf_total |= UVFlag(files)
 
+    # Generate Separable Flags.
+    if separable_flags:
+        broadcast_flags = np.zeros(uvf_total.flag_array.shape).astype(bool)
+        for pol in range(uvf_total.flag_array.shape[2]):
+            fa = uvf_total.flag_array[:, :, pol]
+            # First, flag all frequencies above threshold.
+            for m in range(fa.shape[1]):
+                if np.count_nonzero(~fa[:, m]) / fa.shape[0] >= freq_threshold:
```

Comment:
Why is `fa` negated? This looks like you're counting the number of non-flagged elements?

Reply:
Thanks for catching this bug!